swift/ql/src/queries/Security/CWE-321/HardcodedEncryptionKey.ql

```diff
@@ -0,0 +1,70 @@
+/**
+ * @name Hard-coded encryption key
+ * @description Using hardcoded keys for encryption is not secure, because potential attacker can easiy guess them.
```

Comment:
Did you intend to write `because a potential attacker`? Or maybe `because potential attackers`?

Reply:
I intended the latter. Fixed it now.